### ISSUE TITLE
[SF][Card][A11y]: Support heading level for the title of Card

### ISSUE THREAD

**Issue by ericchan1983** [SAP/ui5-webcomponents#3712]
**Describe the bug**
The heading level is hard-code with aria-level="3". There is no way to assign with other level.

**Isolated Example**

**To Reproduce**

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/1614528/130379441-2865d1fe-46b6-490b-86a9-e46d9abf3fa1.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` version:0.31.14
`@ui5/webcomponents-react` version:0.131.13
Operating System:macOS BigSur
Browser:Chrome 92.0.4515.159 (Official Build) (x86_64)

**Additional context**

**Comment by IlianaB** [comment]
Hello @SAP/ui5-webcomponents-topic-rd I am forwarding this issue to you, as it seems the hard-coded aria-level="3" of the Card Header cannot be changed, and it causes an accessibility error.

**Comment by fifoosid** [comment]
Hi @ericchan1983 

Could you describe what is your use case and why do you need to change the title level?

**Comment by ericchan1983** [comment]
Hi @fifoosid,

The card title is not always aira-level=3, can the web-component provide a setting to config the aria-level, please?

Regards,
Eric Chen

**Comment by fifoosid** [comment]
The issue is fixed, but it won't be downported in 0.31.x branch, since the affected component is not present there!

**Comment by ericchan1983** [comment]
Hi @fifoosid,

Even though the affected component CardHeader is not present in 0.31.x, the issue is also existing in the Card web component. In the 0.31.x, the Card used the heading (string) to set the card title.
Could you also fix this issue on the Card web component to support the heading level on Card title, please?

Regards,
Eric Chen

**Comment by fifoosid** [comment]
Hi @ericchan1983 

As mentioned above, the fix cannot be downported in the 0.31 branch as the ```ui5-card-header``` component is not present there.

However, ariaLevel property is now exposed on ```ui5-card``` level and can be used to achieve the same result. It will be available with the next version from branch 0.31. Keep in mind that once you upgrade the version of ```ui5-webcomponents``` you are using, you will no longer use the ```ui5-card``` property, but the one that is exposed on ```ui5-card-header```

**Comment by ilhan007** [comment]
Fixed with 0.31.18
